docs/README.md

```diff
@@ -1,28 +1,115 @@
-# README
+# Documentation for Unity Catalog
 
-## Serving the documentation with mkdocs
+Unity Catalog uses [**MkDocs**](https://www.mkdocs.org/) to manage and serve its documentation. MkDocs is a simple, static site generator that’s geared towards project documentation and is written in Python.
 
-Create a virtual environment:
+This guide will help you set up the local environment, serve the documentation locally, and deploy it to GitHub Pages.
 
-```sh
+## Setting Up MkDocs for Local Development
+
+To start testing or modifying the documentation locally, follow these steps:
+
+### 1. Install MkDocs
+
+While you may install **MkDocs** directly using `pip`, it is recommended to install it along with other dependencies via the `requirements-docs.txt` file in step 3. This ensures that all necessary dependencies are handled together, especially when working in a virtual environment.
+
+If you prefer to use **pipx**, you can install MkDocs globally with the following command:
+
+   ```bash
```

Comment:
I think you don't need to indent it here. Usually this is done in this documentation within a list to better show where the snippet belongs to (See comment from Denny [here](https://github.com/unitycatalog/unitycatalog/pull/648#discussion_r1811876131)).

Reply:
like would be grateful if you could provide exact lines to change by what.

Reply:
Let me try the suggestion feature (never used it before). Hopefully this helps you.